ghost/i18n/locales/es/portal.json

```diff
@@ -9,168 +9,168 @@
+    "Phone number": "Numero telefónico",
     "Plan": "Plan",
     "Plan checkout was cancelled.": "El pago de plan fue cancelado.",
-    "Plan upgrade was cancelled.": "La actualización de plan fue cancelada.",
-    "Please contact {{supportAddress}} to adjust your complimentary subscription.": "Por favor, ponte en contacto con {{supportAddress}} para ajustar tu suscripción gratuita.",
-    "Please enter {{fieldName}}": "",
+    "Plan upgrade was cancelled.": "La actualización del plan fue cancelada.",
+    "Please contact {{supportAddress}} to adjust your complimentary subscription.": "Por favor contacta a {{supportAddress}} para ajustar tu suscripción gratuita.",
+    "Please enter {{fieldName}}": "Por favor llena lo siguiente {{fieldName}}",
     "Please fill in required fields": "Por favor llena los campos requeridos.",
     "Price": "Precio",
     "Re-enable emails": "Reactivar correos electrónicos",
     "Recommendations": "Recomendaciones",
-    "Renews at {{price}}.": "Se renueva en {{price}}.",
+    "Renews at {{price}}.": "Se renueva a {{price}}.",
     "Retry": "Reintentar",
     "Save": "Guardar",
     "Send an email and say hi!": "¡Envía un correo electrónico y saluda!",
-    "Send an email to {{senderEmail}} and say hello. This can also help signal to your mail provider that emails to and from this address should be trusted.": "¡Envia un correo electrónico a {{senderEmail}} y saluda! Esto también puede ayudar a indicarle a su proveedor de correo que se debe confiar en los correos electrónicos hacia y desde esta dirección.",
+    "Send an email to {{senderEmail}} and say hello. This can also help signal to your mail provider that emails to and from this address should be trusted.": "¡Envia un correo electrónico a {{senderEmail}} y saluda! Esto también puede ayudar a su proveedor de correo que debe confiar en los correos electrónicos hacia y desde esta dirección de correo.",
     "Sending login link...": "Enviando enlace de inicio de sesión...",
     "Sending...": "Enviando...",
     "Show all": "Mostrar todo",
     "Sign in": "Iniciar sesión",
     "Sign out": "Cerrar sesión",
     "Sign up": "Registrarse",
     "Signup error: Invalid link": "Error de registro: Enlace inválido",
-    "Something went wrong, please try again later.": "",
+    "Something went wrong, please try again later.": "Algo no funcionó, por favor intenta mas tarde.",
     "Sorry, that didn’t work.": "Lo siento, eso no funcionó.",
     "Spam complaints": "Quejas de spam",
-    "Start {{amount}}-day free trial": "Comienza el período de prueba gratuito de {{amount}} días",
+    "Start {{amount}}-day free trial": "Comienza el período de prueba gratuito de {{amount}} días gratis",
     "Starting {{startDate}}": "A partir de {{startDate}}",
     "Starting today": "A partir de hoy",
-    "Submit feedback": "Enviar comentarios",
+    "Submit feedback": "Enviar comentarios de retroalimentación",
```

Comment:
This is text for a button, so unless you feel that the shorter version is wrong, perhaps we can revert this change?

Reply:
Oh got it, sure.

Reply:
Did it already.